phala-cloud/confidential-ai/confidential-model/confidential-ai-api.mdx

```diff
@@ -1,42 +1,45 @@
 ---
-description: Secure LLM inference API running in GPU Trusted Execution Environment
-  (TEE) for confidential AI computing.
-title: GPU TEE Inference API
+description: Secure LLM inference API running in GPU TEE for confidential AI computing.
+title: Confidential AI API
 ---
 
 
-Phala Cloud provides LLM inference API by connecting to Redpill. By navigating to **Dashboard**->**GPU TEE API** page, you will see details.
+Phala Cloud provides LLM inference API by connecting to Redpill. By navigating to **Dashboard -> Confidential AI API** page, you will see details.
 
 ## Enable API Service
 
 In order to use the API, **user needs to deposit at least $5 in advance to enable a Redpill account** if your balance is not sufficient. You can head to **Dashboard**->**Billing** page then click **Deposit** button to top-up with your bank card or crypto wallet.
```

Comment:
Avoid mentioning Redpill because it's unnecessary. User just want to use the API